allennlp/service/server_sanic.py

```diff
@@ -8,34 +8,36 @@ def run(port: int) -> None:
     print("Starting a sanic server on port {}.".format(port))
     app = make_app()
     # TODO(joelgrus): make this configurable
-    app.servables = ServableCollection.default()
+    app.predictors = PredictorCollection.default()
```

Comment:
No strong opinion here; if you have a strong opinion, do what you think is best.  It just seemed like having a class that mimics a registry was overkill for what you needed.